group-coordinator/src/test/java/org/apache/kafka/coordinator/group/assignor/OptimizedUniformAssignmentBuilderTest.java

```diff
@@ -245,22 +237,21 @@ public void testFirstAssignmentNumMembersGreaterThanTotalNumPartitions() {
 
     @Test
     public void testValidityAndBalanceForLargeSampleSet() {
-        Map<Uuid, TopicMetadata> topicMetadata = new HashMap<>();
-        for (int i = 1; i < 100; i++) {
+        MetadataImageBuilder metadataImageBuilder = new MetadataImageBuilder();
+        Set<Uuid> topicIds = new HashSet<>();
+        IntStream.range(1, 101).forEach(i -> {
```

Comment:
Is 1 to 101 correct? It may be easier to keep the for loop.

Reply:
Updated it.